client/js/utils/ajax.js

```diff
@@ -2,10 +2,12 @@ import request from 'request';
 import when from 'when';
 
 
+// FIXME: replace with XMLHTTPRequest
```

Comment:
>Lightweight Footprint
> Only 32kB minified and gzipped.

Not that it matters because everyone and their grandma will have it in cache.

```javascript
var jQueryPromise = $.ajax('/data.json');
var realPromise = Promise.resolve(jQueryPromise);
//now just use `realPromise` however you like.
```
Although I didn't find anywhere that actually illuminated the differences between the two. Do you have a link?